src/pages/InfoSec/InfoSec.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Col, Container, Row } from 'react-bootstrap';
+import webgif from '../../assets/CougarCS_Banner.png';
+import { MetaData } from '../../components/Meta/MetaData';
+import './InfoSec.css';
+
+const meta = {
+	title: 'Info-Sec',
+	desc: 'Learn more about CougarCS Web-Dev',
+	url: 'https://cougarcs.com/web-dev',
```

Comment:
change to be accurate to current site.
desc: 'Learn more about CougarCS Info-sec'
url: 'https://cougarcs.com/infosec'